pkgs/servers/monitoring/zabbix/versions.nix

```diff
@@ -1,11 +1,18 @@
 generic: {
+  v60 = generic {
+    version = "6.0.14";
+    sha256 = "sha256-YxrVl12OBxkB/cEvlGR+mV7bTBe6nRi71wLCtZPCzlg=";
+    vendorSha256 = null;
```

Comment:
Should this be `null`?

Reply:
Oh. I'm not familiar enough with our `go` infrastructure off the top of my head to confirm... but the message seems pretty straight forward. Did you test that the specific package builds and runs for that version? If so I would suggest that we're good.